features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,10 @@
+When("I visit the site") do
+    visit root_path
+end
+
+Given("the following articles exist") do |table|
+    table.hashes.each do |article|
+        Article.create(article)
```

Comment:
Use the FactoryBot for creating objects.

Reply:
ok done that